index.js

```diff
@@ -74,9 +74,7 @@ Rapptor.prototype._setupConfig = function() {
   this.config.cwd = this.cwd;
 
   //mongo
-  this.mongoHost = process.env.MONGO_PORT_27017_TCP_ADDR || this.config.mongo.host;
-  this.mongoPort = process.env.MONGO_PORT_27017_TCP_PORT || this.config.mongo.port;
-  this.config.mongo.url = 'mongodb://'+this.mongoHost+':'+this.mongoPort+'/'+this.config.mongo.db;
+  this.config.mongo.url = 'mongodb://'+this.config.mongo.host+':'+this.config.mongo.port+'/'+this.config.mongo.db;
```

Comment:
Maybe, problem is mongo.db is {{name}}-app and would cause the mongo url to be `mongodb://localhost:27017/{{name}}-app`.

Not entirely sure what the best approach for that would be.